docs/developer/plugin-list.asciidoc

```diff
@@ -6,7 +6,7 @@ NOTE:
 
     node scripts/build_plugin_list_docs
 
-  You can update the template within packages/kbn-dev-utils/target/plugin_list/generate_plugin_list.js
+  You can update the template within node_modules/@kbn/dev-utils/target/plugin_list/generate_plugin_list.js
```

Comment:
Hum, I don't think you wanted to change that, did you?

Reply:
I did 😄 `packages/kbn-dev-utils/target/` is not valid anymore synce now the target for the packages are in a temporary arbitrary folder defined by bazel to hold the builds (linked inside kibana with the bazel-bin symlink). As such to keep a similar information as we had before I thought about letting the path of the package in the node_modules (that should be the way we think about our packages in terms of using their code anyway)